src/EditorFeatures/TestUtilities/NavigateTo/AbstractNavigateToTests.cs

```diff
@@ -201,24 +200,6 @@ protected static int CompareNavigateToItems(NavigateToItem a, NavigateToItem b)
     }
 
 #pragma warning disable CS0067
-    public class FirstDocIsActiveDocumentTrackingService : IDocumentTrackingService
-    {
-        private readonly Workspace _workspace;
-
-        public FirstDocIsActiveDocumentTrackingService(Workspace workspace)
-        {
-            _workspace = workspace;
-        }
-
-        public event EventHandler<DocumentId> ActiveDocumentChanged;
-        public event EventHandler<EventArgs> NonRoslynBufferTextChanged;
-
-        public DocumentId GetActiveDocument()
-            => _workspace.CurrentSolution.Projects.First().DocumentIds.First();
-
-        public ImmutableArray<DocumentId> GetVisibleDocuments()
-            => ImmutableArray<DocumentId>.Empty;
```

Comment:
why remove all this.  IIRC this was to actually test out navto scenarios.

Reply:
Commented in the commit: this is testing a case where a document is active, but not visible. That's not possible. :smile:

Reply:
haha.  Ok :)   Seems reasonable :D